test/test-async.js

```diff
@@ -3,7 +3,7 @@
  *
  *
  ** */
-process.env.TZ = 'America/San_Francisco';
+process.env.TZ = 'Europe/Zurich';
```

Comment:
Why do you need to change to "Europe/Zurich" here?

Reply:
as full day events don't have a timezone, the problem only occurs when the current time is east of UTC. 

I couldn't find a way to change the timezone just for this test..

Reply:
Have you tried `moment.tz.setDefault(String)`? See:

https://momentjs.com/timezone/docs/#/using-timezones/default-timezone/

Reply:
how do you do that in testcase was the problem

Reply:
what about this? Did you test this?

```diff
--- a/test/test.js
+++ b/test/test.js
@@ -3,18 +3,18 @@
  *
  *
  ** */
-process.env.TZ = 'America/San_Francisco';
-
 const assert = require('assert');
 const vows = require('vows');
 const _ = require('underscore');
 const ical = require('../node-ical');
+const moment = require('moment-timezone');
 
 vows
   .describe('node-ical')
   .addBatch({
     'when parsing test1.ics (node conferences schedule from lanyrd.com, modified)': {
       topic() {
+        moment.tz.setDefault('America/San_Francisco');
         return ical.parseFile('./test/test1.ics');
       },
```

Reply:
No. didn't know that you could do that.. I looked all over.. don't know vows, nothing in the doc implies u can do that

Reply:
I am not sure if this works or not because it could be that the setDefault only works for the moment object, but I prefer try&error here. So see if you can change it accordingly and if it works. then use that method to change the timezone only for the new testcase.